modules/claims_api/app/mailers/claims_api/submission_report_mailer.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class SubmissionReportMailer < ApplicationMailer
+    RECIPIENTS = %w[
+      alex.wilson@oddball.io
+      austin.covrig@oddball.io
+      emily.goodrich@oddball.io
+      jennica.stiehl@oddball.io
+      kayla.watanabe@adhocteam.us
+      matthew.christianson@adhocteam.us
+      rockwell.rice@oddball.io
+    ].freeze
+
+    def build(date_from, date_to, submissions = nil) # rubocop:disable Metrics/MethodLength
+      @date_from = date_from.in_time_zone('Eastern Time (US & Canada)').strftime('%a %D %I:%M %p')
+      @date_to = date_to.in_time_zone('Eastern Time (US & Canada)').strftime('%a %D %I:%M %p')
+      @data = { month: {}, total: {} }
+      submissions = ClaimsApi::ClaimSubmission.where(created_at: @from..@to) if submissions.nil?
+
+      submissions.pluck(:claim_type).uniq.sort.each do |kind|
+        @data[:month][kind] = {}
+        subs = submissions.select { |sub| sub[:claim_type] == kind }
+        subs.pluck(:consumer_label).uniq.sort.each do |label|
+          @data[:month][kind][label] = subs.select { |sub| sub[:consumer_label] == label }.size
+        end
+      end
+
+      submissions = ClaimsApi::ClaimSubmission.all
+      submissions.pluck(:claim_type).uniq.sort.each do |kind|
```

Comment:
Can we cache the total and add the new month each time rather than calling all from the DB every time?